src/index.js

```diff
@@ -1,8 +1,9 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
```

Comment:
We arent using `useEffect` in this file?

Reply:
Whoops... I thought linting would have caught that? Good spot

Reply:
Oh it did :p my bad